eixampledb-core/src/main/java/com/eixampledb/core/api/EixampleDbEntry.java

```diff
@@ -3,12 +3,12 @@
 public class EixampleDbEntry {
 
     private final String key;
-    private final String value;
+    private final Object value;
     private final long creationTimestamp;
     private final long lastupdateTimestamp;
-    private final int type;
+    private final ValueType type;
 
-    public EixampleDbEntry(String key, String value, long creationTimestamp, long lastupdateTimestamp, int type) {
+    public EixampleDbEntry(String key, Object value, long creationTimestamp, long lastupdateTimestamp, ValueType type) {
```

Comment:
Change from int representation of type to enum representation. It is much more concise. Also we can store the values as objects, so if a String as a String and a Long as a Long. So later we don't have to parse on write nor parse on read.